contracts/interfaces/IStableDebtToken.sol

```diff
@@ -14,17 +14,17 @@ interface IStableDebtToken is IInitializableDebtToken {
    * @dev Emitted when new stable debt is minted
    * @param user The address of the user who triggered the minting
    * @param onBehalfOf The recipient of stable debt tokens
-   * @param amount The amount minted (user entered amount + balance increase from interest)
-   * @param currentBalance The current balance of the user
-   * @param balanceIncrease The increase in balance since the last action of the user
+   * @param value The amount minted (user entered amount + balance increase from interest)
+   * @param currentBalance The balance of the user based on the previous balance and balance increase from interest
+   * @param balanceIncrease The increase in balance since the last action of the user 'onBehalfOf'
    * @param newRate The rate of the debt after the minting
    * @param avgStableRate The next average stable rate after the minting
    * @param newTotalSupply The next total supply of the stable debt token after the action
    **/
   event Mint(
     address indexed user,
     address indexed onBehalfOf,
-    uint256 amount,
+    uint256 value,
```

Comment:
The subgraph will need to differentiate pre and post patch versions

Reply:
This doesn't seem reasonable only due to a name change. Unless the subgraph breaks anyway or there is some major reason

Reply:
The subgraph will need to differentiate either way. However, this change will affect the subgraph logic so I may be interesting to revert it back. Thoughts @defispartan ?

Reply:
Last month we implemented a [fix](https://github.com/aave/protocol-subgraphs/pull/84/commits/5ac8df2e39777ebfbdb88b3bc409e43032c18a9a) for the current V3 subgraphs, so now these subgraphs correctly handle mint and burn events. With this change we would need to add versioning throughout each V3 subgraph to say:
```
if(blockNumber > ...){
	// use updated mint/burn logic
	// use updated naming: event.params.value
} else {
	// use current mint/burn logic
	// use current naming: event.params.amount
}
```
Since we have a working version already and handling this would be quite ugly, I would be in favor of reverting the event change

Reply:
Makes sense to me. Reverting the change in order to make the subgraph's logic less painful